IdentityCore/src/oauth2/MSIDBrokerResponse.m

```diff
@@ -72,4 +72,17 @@ - (NSString *)oauthErrorCode
     return _urlForm[@"code"];
 }
 
+- (BOOL)isAccessTokenCompromised
```

Comment:
Some comments here:
1. Compromised sounds like somebody stole it :) Can be change its name to something more appropriate, e.g. invalid?
2. I'd prefer saveTokensWithBrokerResponse to accept additional parameter like "saveRefreshTokenOnly" and keep the logic below in ADAL only. As MSAL will never need it, I don't see a reason to have it here.

Reply:
done